src/index.ts

```diff
@@ -388,7 +388,16 @@ export async function init(config: IClientConfig): Promise<EppoClient> {
       skipInitialPoll: skipInitialRequest,
     };
     instance.setConfigurationRequestParameters(requestConfiguration);
-    instance.setEventDispatcher(newEventDispatcher(apiKey));
+
+    // e.g. newDefaultEventDispatcher throws an error if it cannot parse a valid event ingestion URL from the SDK key
+    try {
+      instance.setEventDispatcher(newEventDispatcher(apiKey));
```

Comment:
fixed in https://github.com/Eppo-exp/js-sdk-common/pull/143 to no longer throw
I'd vote removing this try/catch since it should never throw after that change

Reply:
Sounds good. I'll close this out